### PR TITLE
About module

### PR DESCRIPTION
**Pull request by surma**
Also contains some generic module styling and setup.

### CODE REVIEW

File: src/_includes/about/index.njk [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<link rel="stylesheet" href="confboxAsset({{css}})">
+<section class="{% className css, 'about-module' %}">
+  <h1>About</h1>
+  <p>
+    <strong>Chrome Dev Summit</strong> is back! Join us for our annual developer conference at the <a href="https://goo.gl/maps/bUAF6Bore8khBojZ8">Yerba Buena Center for the Arts</a> in San Francisco, CA on <strong>11—12 November 2019</strong>.
+  </p>
+  <p>
+    Over two days, you’ll learn about the latest tools and updates coming to the web platform and hear directly from the <a href="https://www.google.com/chrome/">Chrome</a> product engineering teams.
+  </p>
+  <iframe class="{% className css, 'about-media' %}" src="https://www.youtube.com/embed/nImH_dq1NpM" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (surma) [inline src/_includes/about/index.njk]: We probably wanna lazy-load this, but for now this will do.

File: src/_includes/module.css [hunk 2]
```diff
@@ -0,0 +1,64 @@
+.module {
+  background-color: var(--white);
+  padding: var(--vertical-margin) var(--side-margin);
+
+  &:nth-of-type(2n + 0) {
+    background-color: var(--offwhite);
+  }
+  > h1 {
```
Comment (jakearchibald) [inline src/_includes/module.css]: Should be Google sans.

File: src/_includes/layouts/main/style.css [hunk 3]
```diff
@@ -53,12 +53,16 @@ html {
   font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen,
     Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
   font-size: 21px;
+  line-height: 2em;
```
[the remark this answers is not in the document]
Comment (surma) [reply src/_includes/layouts/main/style.css]: Let’s leave it for now and get back to this when it becomes a problem?

**Comment by jakearchibald** [comment]
Let's land this so I can merge it into my stuff. I've filed https://github.com/GoogleChrome/devsummit/issues/127 for a couple of issues.

**Pull request merged**